### ISSUE TITLE
Access request for test-equinix_esxi-ubuntu2204-x64-1

### ISSUE THREAD

**Issue by sophia-guo** [adoptium/infrastructure#3495]
**NOTE: THIS ISSUE SHOULD NOT BE CLOSED BY THE ORIGINATOR IF ACCESS IS GRANTED**.
When the access is no longer needed please add a comment and a member
of the infrastructure team will revoke it and close the issue.

Required access level (Delete as appropriate). Note that you should only
request the minimum level that is required to solve your problem

- [ ] Non-privileged
- [ ] jenkins user
- [ ] root/Administrative
- [ ] other (Please specify):

System for which access is needed:

see https://github.com/adoptium/infrastructure/issues/3490

**Comment by sophia-guo** [comment]
@sxa

**Comment by sxa** [comment]
Access is granted to 145.40.115.44

**Comment by sxa** [comment]
@sophia-guo Have you been able to do the diagnosis you need on this machine? It's likely to be decommissioned this week so I want to make sure you don't lose anything you might have on there.

**Comment by sophia-guo** [comment]
Yes, it helps. I would still need it to do more testing. Hopefully by the end of this week I can finish the testing.

**Comment by sxa** [comment]
@sophia-guo Are you finished with this machine now? I'm looking to start deactivating the ESXi host

**Comment by sophia-guo** [comment]
yes, thanks! @sxa
